[tutorials] Working with Lists tutorial (#730)
lit/lit.dev: packages/lit-dev-content/site/tutorials/content/working-with-lists/06.md

> There are several ways of adding event listeners on each item getting rendered

Reviewer: ```suggestion
There are several ways of adding event listeners to list items
```

> from a list, but care must be taken to ensure the correct information is made

Reviewer: ```suggestion
, but care must be taken to ensure the correct information is made
```
